src/rtccertificate.cc

```diff
@@ -86,3 +96,35 @@ NAN_GETTER(RTCCertificate::GetFingerprints) {
 
   info.GetReturnValue().Set(array);
 }
+
+NAN_METHOD(RTCCertificate::ToPEM) {
+  METHOD_HEADER("RTCCertificate", "toPEM");
+  UNWRAP_OBJECT(RTCCertificate, object);
+
+  rtc::RTCCertificatePEM _pem = object->_certificate->ToPEM();
+
+  Local<Object> pem = Nan::New<Object>();
+  pem->Set(LOCAL_STRING(kPrivateKey), LOCAL_STRING(_pem.private_key()));
+  pem->Set(LOCAL_STRING(kCertificate), LOCAL_STRING(_pem.certificate()));
+
+  info.GetReturnValue().Set(pem);
+}
+
+NAN_METHOD(RTCCertificate::FromPEM) {
+  METHOD_HEADER("RTCCertificate", "fromPEM");
+
+  ASSERT_SINGLE_ARGUMENT;
+  ASSERT_OBJECT_ARGUMENT(0, pem);
+  ASSERT_OBJECT_PROPERTY(pem, kPrivateKey, privateKeyVal);
+  ASSERT_PROPERTY_STRING(kPrivateKey, privateKeyVal, privateKey);
+
+  ASSERT_OBJECT_PROPERTY(pem, kCertificate, certificateVal);
+  ASSERT_PROPERTY_STRING(kCertificate, certificateVal, certificate);
+
```

Comment:
Some tests should be added about the arguments count and types, it might sound paranoid, but I'll handle this myself if you want.